event/consumer_test.go

```diff
@@ -102,7 +101,7 @@ func TestConsume(t *testing.T) {
 			})
 
 			Convey("Release is called on the consumer", func() {
-				So(len(mockConsumer.ReleaseCalls()), ShouldEqual, 1)
+				So(len(message.ReleaseCalls()), ShouldEqual, 1)
```

Comment:
This is perfectly fine and correct and you can keep this as it is but just as an FYI there is a `ShouldHaveLength` assertion:

`So(message.ReleaseCalls(), ShouldHaveLength, 1)`

It does exactly the same thing but reads more like English and makes the code a bit easier to read.